script.js

```diff
@@ -54,45 +54,89 @@ window.addEventListener('click', function(event) {
-    "cv": "You can download my CV by clicking the 'Download CV' button in the navigation menu. It contains more detailed information about my experience and education.",
-    "portfolio": "You can view my portfolio and projects on GitHub and LinkedIn. I have worked on creating my porfolio profile so far",
-    "projects": "Portfolio Profile bu currently working on a new project",
-    "github": "Check out my GitHub profile [here](https://github.com/Sisonke10-byt) for my latest repositories and projects.",
-    "linkedin": "Connect with me on LinkedIn [here](https://www.linkedin.com/in/sisonkesamkele).",
-    "default": "Sorry, I didn't understand that. Can you ask something else?",
+    "greeting": {
+        keywords: ["hello", "hi", "hey", "howdy", "good morning", "good evening"],
+        response: "Hi there! How can I help you today?"
+    },
+    "how_are_you": {
+        keywords: ["how are you", "how's it going", "how are things", "what's up"],
+        response: "Great! How can I help you?"
+    },
+    "about_me": {
+        keywords: ["about you", "who are you", "tell me about yourself"],
+        response: "I am Sisonke Boyilani, a recent graduate with a diploma in Information Technology Management, currently working as a DevOps Intern."
+    },
+    "skills": {
+        keywords: ["skills", "what can you do", "what are your skills"],
+        response: "I am proficient in HTML, CSS, JavaScript, and more. I have strong communication and problem-solving skills, as well as technical expertise in DevOps practices."
+    },
+    "education": {
+        keywords: ["education", "where did you study", "your background"],
+        response: "I graduated with a Diploma in Information Technology Management from Rosebank College."
+    },
+    "contact": {
+        keywords: ["contact", "reach me", "how to contact"],
+        response: "You can reach me at sisonkeboyilani@capaciti.org.za or call +27 765759788."
+    },
+    "cv": {
+        keywords: ["cv", "resume", "curriculum vitae", "download cv"],
+        response: "You can download my CV by clicking the 'Download CV' button in the navigation menu."
+    },
+    "portfolio": {
+        keywords: ["portfolio", "projects", "portfolio profile", "what have you worked on"],
+        response: "You can view my portfolio and projects on GitHub and LinkedIn. I'm currently working on a new project."
+    },
+    "github": {
+        keywords: ["github", "github profile", "repositories"],
+        response: "Check out my GitHub profile [here](https://github.com/Sisonke10-byt) for my latest repositories and projects."
+    },
+    "linkedin": {
+        keywords: ["linkedin", "connect with me", "social media"],
+        response: "Connect with me on LinkedIn [here](https://www.linkedin.com/in/sisonkesamkele)."
+    },
+    "default": {
+        keywords: [],
+        response: "Sorry, I didn't understand that. Can you ask something else?"
+    }
 };
```

Comment:
This is great work, your responses are more detailed.

Reply:
Thank you @Andro-crt-prog

Reply:
Looks professional, great work.